src/scripts/bases.php

```diff
@@ -5,7 +5,18 @@
   exit(0);
 }
 
-require_once ('/var/www/fbctf/vendor/autoload.php');
+require_once ('/var/www/fbctf/src/Db.php');
```

Comment:
I think we better use:
```
require_once (__DIR__.'/../Db.php');``` 
require_once (__DIR__.'/../Utils.php');
require_once (__DIR__.'/../Model.php');
require_once (__DIR__.'/../Importable.php');
require_once (__DIR__.'/../Exportable.php');
require_once (__DIR__.'/../Level.php');
require_once (__DIR__.'/../Link.php');
require_once (__DIR__.'/../Team.php');
require_once (__DIR__.'/../Configuration.php');
require_once (__DIR__.'/../ScoreLog.php');
require_once (__DIR__.'/../Control.php');
require_once (__DIR__.'/../MultiTeam.php');
```
to avoid the hardcoded path, and I think that is the main problem now.